suzieq/cli/sqcmds/DeviceCmd.py

```diff
@@ -51,7 +50,7 @@ def __init__(
         self.lvars = {
             'os': os.split(),
             'version': version.split(),
-            'status': status,
+            'status': status.split(),
             'model': model,
```

Comment:
We are not accepting multiple values in the `model` field

Reply:
Nice try, but no dice :) The model split is a little more complicated because model names can have spaces inside them. So the split happens a little above, if you read it more carefully :-p

Reply:
But I did find that my regex didn't work correctly. I used shlex split to handle this case. See the code and additional tests that I've added

Reply:
I think we still have a problem if we search for `My device model` this is the array we get `['My', 'device', 'model']`, which I think is fine, since coherent with the rest. So let's add quotes, so we search for `'My device model'`, since Nubia strips the quotes the result is equal to the previous one. Let's now try to search for multiple models `'My device model' 'model-x'`, this is the string we get `My device model' 'model-x` and this is the array we get `['My', 'device', 'model model-x']`, which is completely wrong. Since trailing and leading quotes are stripped, distinguishing between space as separator or as space of the model string is not simple at all. Some possible solution I could come up with are:
1. Always add trailing and leading quotes, but at that point we are forced to quotes when defining multiple models in the filter
2. Using a different separator which is not the space, but this won't be coherent with the rest
3. Users can search for a single model at a time

Reply:
So, he way to search for multiple models is: model=" 'my device model' vm". We'll have to document this somewhere. I agree that this is ridiculous, but we can't do anything until we fix nubia. I don't like 2, and we could live with 3.